recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "great-expectations" %}
-{% set version = "0.8.2" %}
-{% set sha256 = "ac28c43976956178a9968f9eb599cba14ffa13bc0c723e8ba915af26711e459e" %}
+{% set version = "0.8.2.post0" %}
```

Comment:
Is this an intended release version?

Reply:
Yes, there was a small packaging error in the 0.8.2 release that caused css files to be excluded

Reply:
Thanks for clarifiying.